third_party/istio-1.1.13/values.yaml

```diff
@@ -39,7 +39,10 @@ gateways:
     replicaCount: 2
     autoscaleMin: 2
     autoscaleMax: 4
-    resources: {}
+    resources:
+      requests:
+        cpu: 250m
+        memory: 256Mi
```

Comment:
Intentional change?

Reply:
Yes, because the default is 10m which is abysmally low